### ISSUE TITLE
max images per row property 

### ISSUE THREAD

**Issue by OriAmir** [benhowell/react-grid-gallery#86]
There is some property of 'max rows' but not exist option of **max images per row.**.. let's say i want max 2 or 3 images in a row. I think it's can be really useful. 

Another question-there is some option to get thumbnail in relative to the image .. let's say my image is 2000x5000 , so i want the thumbnail will be good enough , when i try just put thumbnailWidth: 2000/10 and thumbnailHeight: 5000/10 I get the original picture without the calculate.

thanks a lot!

**Comment by benhowell** [comment]
Images per row needs to be dynamic in order to create full rows. 

I don't understand your question about thumbnailWidth and thumbnailHeight. Perhaps you can post an example of what you're trying to do?

**Comment by OriAmir** [comment]
@benhowell thanks for your answer .
So we can't somehow decide image for row ? i don't understand your answer about that.

about thumbnailWidth/Height I will explain: 
My gallery image is on different sized , some is more width and some are more height , I want that the thumbnail will looks good and want to give him size that have the same proportions like the real image size.
So I say if for example my picture is 2000x5000 (2000 width,5000 height)  and another picture is 3000x1500 how can I choose right values for thumbnailWidth and thumbnailHeight that looks good ?

**Comment by benhowell** [comment]
It's a justified gallery, so the whole point is to have justified rows with images filling the entire row.

Thumbnails need to either be separate images and you use the actual width and height of those images for thumbnailWidth and thumbnailHeight, or, you use your original images and use the actual width and height of those images for thumbnailWidth and thumbnailHeight. So if the image specified by thumbnail is actually 2000x5000, then they are the values you use.

I would advise against using the original images as your thumbnails as the page size (data) would be huge.

**Comment by OriAmir** [comment]
@benhowell thanks  , but I don't really understand the meaning of 'thumbnailHeight' here , because the height is always the same for all images ('rowHeight' property ) , I Can't set thumbnail in different height sized?

**Comment by benhowell** [comment]
No. The rows are always equal height and the thumbnails within them will be of equal height. 

As I've said before, the thumbnail height is the actual height of the thumbnail image. Row height is the height of each row. Thumbnails will be scaled to fit the row height.
